dm/dumpling/dumpling.go

```diff
@@ -125,6 +125,11 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 		m.mu.Lock()
 		m.core = dumpling
 		m.mu.Unlock()
+
+		failpoint.Inject("dumpSleepStage", func() {
+			time.Sleep(2 * time.Second)
```

Comment:
i think this failpoint is not steable enough, for example ,if test scripts running slow(more than 2s), test case will failed

so maybe reuse current failpoint `dumpUnitProcessForever`